scripts/fonts/chikarego2-polish.py

```
@@ -0,0 +1,90 @@
+            width = f[base].width
+    if width is not None:
+        g.width = width
+    return g
+
+# pixel rows: x-height top = 7px (448), cap top = 9px (576), baseline = 0
+def acute_lc(g, leftcol):              # 3px staircase, bottom row 8 (y512)
+    px(g, leftcol, 8); px(g, leftcol+1, 9); px(g, leftcol+2, 10)
+def acute_uc(g, leftcol):              # 2px staircase, bottom row 10 (y640) — matches native Ó
+    px(g, leftcol, 10); px(g, leftcol+1, 11)
+def acute_lc_short(g, leftcol):        # 2px staircase (shorter), bottom row 8
+    px(g, leftcol, 8); px(g, leftcol+1, 9)
```

Ruff (0.15.17)

[error] Multiple statements on one line (semicolon)

(E702)